client/test/integration/jest.setup.ts

```diff
@@ -0,0 +1,7 @@
+import '@testing-library/jest-dom';
+import './__mocks__/module_mocks';
```

Comment:
How are the module mocks of unit and integration tests linked? Is it possible to separate them using the jest configuration itself?

Reply:
> Is it possible to separate them using the jest configuration itself?

This is exactly how it is done.